package/base-files/files/etc/init.d/led

```diff
@@ -61,13 +68,14 @@ load_led() {
 		echo $trigger > /sys/class/leds/${sysfs}/trigger 2> /dev/null
 		ret="$?"
 
-		[ $default = 1 ] &&
-			cat /sys/class/leds/${sysfs}/max_brightness > /sys/class/leds/${sysfs}/brightness
+		[ $default = 1 ] || [ ! "$trigger" = "none" ] &&
+			echo "$brightness" > /sys/class/leds/${sysfs}/brightness
 
 		[ $ret = 0 ] || {
 			echo >&2 "Skipping trigger '$trigger' for led '$name' due to missing kernel module"
 			return 1
 		}
+
```

Comment:
while this change probably is good, at the least put it in its own commit, or just leave it out (for now) ...

Reply:
I'm not 100% sure to understand. Are you talking about the blank line or the lines 71-72?

Reply:
the blank line in line 78 :)

Reply:
Sure, I'll revert it, no problem!

Reply:
done